_posts/2013-04-06-developing-addons-and-blueprints.md

```diff
@@ -594,3 +594,68 @@ running `ember init` in your project root.
 For a good walkthrough of the (recent) development of a real world
 addon, take a look at:
 [Creating a DatePicker Ember CLI addon](http://edgycircle.com/blog/2014-creating-a-datepicker-ember-addon)
+
+### In-Repo Addons
+
+Addons sepecific to your project can be created inside your repo and are
+generated in the projects `lib` directory in a folder with the name of
+the in-repo addon, e.g. `/lib/in-repo-addon-name` and follow the same
+file structure as a normal *addon*.
+
+Some advantages of using an in-repo-addon, instead of an addon outside of
+your application (repo), are:
+
+- Sandbox for developing a feature that you may want to share as an
+  addon in the future
+- Having all the benefits of addon isolation but without the need to
+  publish or `npm link`
+
+Use `in-repo-addon` argument with the `ember generate` command:
+
+{% highlight bash %}
+ember generate in-repo-addon in-repo-addon-name
+{% endhighlight %}
+
+(Replace `in-repo-addon-name` with the name of your addon.)
+
+Shorthand arguments `-ir` and `-in-repo` can be used in place of `--in-repo-addon`.
```

Comment:
This should be clarified that the shorthands are for generating blueprints into a target in-repo-addon, i.e.:
`ember g component foo-bar -ir my-in-repo-addon`